### PR TITLE
feat: removed forta, fixed telegram markdown

### PR DESCRIPTION
**Pull request by sergeyWh1te**
## Summary

1. Removed forta
2. fixed telegram markdown
3. fixed telegram, discord max msg size

### CODE REVIEW

File: cmd/worker/main.go [hunk 1]
```diff
@@ -132,43 +112,16 @@ func main() {
 		worker.WithFindingConsumer(services.OpsGenia, `Protocol_OpGenia_Consumer`, registry.OnChainAlerts, OpsGenie),
```
Comment (TheDZhon) [inline cmd/worker/main.go:112]:
```suggestion
		worker.WithFindingConsumer(services.OpsGenie, `Protocol_OpGenie_Consumer`, registry.OnChainAlerts, OpsGenie),
```

**Review by TheDZhon — APPROVED** [review]
:+1:

File: internal/app/server/server.go [hunk 2]
```diff
@@ -72,8 +71,8 @@ func (a *App) RunHTTPServer(ctx context.Context, g *errgroup.Group, appPort uint
 func (a *App) RegisterHTTPRoutes(r chi.Router) {
 	a.RegisterMiddleware(r)
 
-	alertsH := alerts.New(a.Logger, a.Metrics, a.natsClient, a.env.FortaAlertsTopic)
-	r.Post("/alerts", alertsH.Handler)
+	// alertsH := alerts.New(a.Logger, a.Metrics, a.natsClient, a.env.FortaAlertsTopic)
+	// r.Post("/alerts", alertsH.Handler)
```
[the remark this answers is not in the document]
Comment (sergeyWh1te) [reply internal/app/server/server.go:75]: removed

File: internal/pkg/notifiler/telegram_test.go [hunk 3]
```diff
@@ -22,6 +22,143 @@ func intPtr(i int) *int {
 	return &i
 }
 
+const EugineOnegin = `
+«Мой дядя самых честных правил,
```
[the remark this answers is not in the document]
Comment (sergeyWh1te) [reply internal/pkg/notifiler/telegram_test.go:26]: removed

File: internal/pkg/notifiler/format_alert.go [hunk 4]
```diff
@@ -45,3 +46,41 @@ func shortenHex(input string) string {
 	}
 	return fmt.Sprintf("x%s...%s", input[2:5], input[len(input)-3:])
 }
+
+func EscapeMarkdownV1(input string) string {
```
[the remark this answers is not in the document]
Comment (sergeyWh1te) [reply internal/pkg/notifiler/format_alert.go:50]: Added

File: internal/pkg/notifiler/telegram.go [hunk 5]
```diff
@@ -32,29 +32,32 @@ func NewTelegram(botToken, chatID string, httpClient *http.Client, metricsStore
 	}
 }
 
+const maxTelegramMessageLength = 4096
+const warningTelegramMessage = "Warn: Msg >=4096, pls review description message"
```
Comment (TheDZhon) [inline internal/pkg/notifiler/telegram.go:36]:
```suggestion
const warningTelegramMessage = "Warn: msg.length >= 4096, please shorten the description message"
```

**Review by TheDZhon — APPROVED** [review]
GG :+1:

**Pull request merged**